test/dynamo/test_minifier.py

```diff
@@ -104,6 +104,8 @@ def inner():
     def test_at_aot(self):
         self._test_around_aot(True)
 
+    # Minifier might be busted on this branch
```

Comment:
there are some minifier hacks on the branch

Reply:
Yea